ballista/rust/core/src/serde/logical_plan/to_proto.rs

```diff
@@ -407,6 +407,7 @@ impl From<&DataType> for protobuf::arrow_type::ArrowTypeEnum {
                     fractional: *fractional as u64,
                 })
             }
+            _ => ArrowTypeEnum::None(EmptyMessage {}),
```

Comment:
Just a temporary workaround, but need to add the Map type to the proto definitions.